tools/tools.go

```diff
@@ -372,8 +379,13 @@ func EnvmanRun(envstorePth, workDirPth string, cmdArgs []string, timeout time.Du
 		errWriter = outWriter
 	}
 
+	inReader = os.Stdin
+	if stdInPayload != nil {
+		inReader = bytes.NewReader(stdInPayload)
```

Comment:
IMO we can go with this solution for now, but we need to keep in our mind: this way plugin having payload will see our bytes.NewReader on stdin, which blocks you to create plugins, with payload and reading directly from stdin (like an interactive tool, which prompts for input)

Reply:
We agreed to keep this this way